<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="editorsLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="startLabel">
          <property name="text">
           <string>Секунды</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="timerBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Слов в минуту</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="wordsBox"/>
        </item>
        <item>
         <widget class="QLabel" name="cntrLabel">
          <property name="text">
           <string>Точность</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="accuracyBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="startbutton">
          <property name="text">
           <string>Старт/Стоп</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="langBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Немецкий</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Французский</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Арабский</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Китайский</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Белоруссккий</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Иврит</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetbutton">
          <property name="text">
           <string>Использовать дефолтный текст</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openbutton">
          <property name="text">
           <string>Открыть файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="aimTextEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="checkTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
